grass7/vector/v.class.mlR/v.class.mlR.html

```diff
@@ -56,6 +60,33 @@ <h2>DESCRIPTION</h2>
 tunegrids="{'svmRadial': 'sigma=c(0.01,0.05,0.1), C=c(1,16,128)', 'rf': 'mtry=c(3,10,20)'}"
 </pre></div>
 
+<p>Tuning is potentially very time consuming. Using only a subset of the training
+data for tuning can thus speed up the process significantly, without losing 
+much quality in the tuning results. For training, depending on the number of 
+features used, some R functions can reach their capacity limit.
+The user can, therefore, define a maximum size of samples per class both for 
+tuning (<em>tuning_sample_size</em>) and for training 
+(<em>training_sample_size</em>).
+
+<p>Classifying using too many features (i.e. variables describing the objects 
+to be classified) as input can have negative effects on classification accuracy
+(Georganos et al, 2018). The module therefore provides the possibility to run a
+feature selection algorithm on the training data in order to identify those 
+features that are the most efficient for classification. Using less features
+also speeds up the tuning, training and classification processes. To activate 
+feature selection, the user has to set the <em>max_features</em> parameter to 
+the maximum number of features that the model should select. Often, less than 
+this maximum will be selected. The method used for feature selection is recursive 
+feature elimination based on a random forest model. Note thus that feature 
+selection might sub-optimal for other classifiers, notably non tree-based.
+
+<p>The module can be run only for tuning and training a model, but without 
+any classification (<em>-t flag</em>). Any trained model can be saved to a
+file (<em>output_model_file</em>) which can then later be read into the module
+for a specific classification run (<em>input_model_file</em>). This can be 
```

Comment:
why specific classification run? I would replace it for: prediction step to obtain a classified map or so...